# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/stream/tentative/close.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/stream/tentative/close.any.js.ini
--- a/testing/web-platform/meta/websockets/stream/tentative/close.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/websockets/stream/tentative/close.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -344,10 +344,7 @@
 
 [close.any.serviceworker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (processor == "x86") and not debug: ["OK", "TIMEOUT"]
   [close code should be sent to server and reflected back]
     expected: FAIL
 
@@ -435,10 +432,7 @@
 
 [close.any.sharedworker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (processor == "x86") and not debug: ["OK", "TIMEOUT"]
   [close code should be sent to server and reflected back]
     expected: FAIL
 
@@ -698,10 +692,7 @@
 
 [close.any.worker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (processor == "x86") and not debug: ["OK", "TIMEOUT"]
   [close code should be sent to server and reflected back]
     expected: FAIL
 
@@ -789,10 +780,7 @@
 
 [close.any.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and debug: ["TIMEOUT", "OK"]
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if processor == "x86": ["OK", "TIMEOUT"]
   [close code should be sent to server and reflected back]
     expected: FAIL
 